syntax = "proto3";

package export;

service ImageExportService {
  rpc ExportImages (ExportImagesRequest) returns (stream ImageExportResponse);
}

message ExportImagesRequest {
  string author = 1;
  string tag = 2;
}

message ImageExportResponse {
  int32 id = 1;
  string title = 2;
  string image = 3;
  string author = 4;
  repeated string tags = 5;
  string created_at = 6;
}
